src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -3556,7 +3402,7 @@ private void handlePostGenericSubObj(RoutingContext ctx, String subObj) {
       switch (status) {
         case 201:
           contentType = APPLICATION_JSON;
-          body = JsonObject.mapFrom(ctx.getBodyAsJson().mapTo(getSubObjClass(subObj))).put(ID, UUID.randomUUID().toString());
+          body = JsonObject.mapFrom(ctx.getBodyAsJson().mapTo(getSubObjClass(subObj)));
```

Comment:
Alert object will still need to return a UUID , so we should be returning a generated UUID